CMakeLists.txt

```diff
@@ -105,9 +105,7 @@ if("sgx" IN_LIST COMPILE_TARGETS)
                        sss.enclave ccf_endpoints.enclave ccf_kv.enclave
   )
 
-  if(ENABLE_HTTP2)
-    target_link_libraries(ccf.enclave PUBLIC nghttp2.enclave)
-  endif()
+  target_link_libraries(ccf.enclave PUBLIC nghttp2.enclave)
```

Comment:
Note: `nghttp2` is now always part of the TCB.

Reply:
Hm. Should we leave it optional while it's still experimental? I think I remember that there was a problem with that, so maybe that's too much of a hassle.

Reply:
Because we've now cut 2.x and we're not planning to backport these changes, I think it's OK to have this on by default (still under a runtime flag!) as we'll aim to have this out for 3.x anyway? I'm happy to leave it under a compile-time flag if there are other reasons I'm missing.

Reply:
As discussed, we will link `nghttp2` in on `main` but won't backport HTTP/2 support to 2.x.